### ISSUE TITLE
post-processing for instance segmentation

### ISSUE THREAD

**Issue by geekfeiw** [TheCodez/pytorch-box2pix#1]
This is a great project. 

Could you upload code of the post-processing for instance segmentation? Thank you

**Comment by TheCodez** [comment]
@geekfeiw Thanks for your interest in the project.

The instance segmentation step will hopefully come at some point. I'm currently having a hard time understanding the description from the paper, so if someone has some ideas please don't hesitate to share them here.

Until then there are some more task that need to be done before this project can be used, namely: mAP metric and fixing errors in the training script to get it running.

**Comment by geekfeiw** [comment]
I find a similar project, and want to share with you. https://github.com/dbolya/yolact
